example-config.yml

```diff
@@ -338,12 +338,21 @@ itinerary:
 ### The nested structure should be the same as the language files under the i18n folder.
 ### You can also customize OTP error messages for itinerary searches based on OTP HTTP codes.
 ### A separate message can be set for each language or locale if necessary.
+### All available languages should have an entry with at least a name identifier in this section so that
+### they may be rendered in the locale selector. 
+### Language display icons (flags) should be configured in config.js
+# LocalizationEnabled also provides a method to toggle the new localization functionality, which requires 
```

Comment:
This should probably be `localeSelectorEnabled`

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/525/commits/50614e16b8a4687cb376cdba79923c00c2e76071